target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -572,7 +625,8 @@ TARGET_DEVICES += DSL275XB-D1
 
 define Device/DVAG3810BN
   $(Device/bcm63xx)
-  DEVICE_TITLE := D-Link DVA-G3810BN/TL
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := DVA-G3810BN/TL
```

Comment:
Alternate names "DVA-G3810BN" and "DVA-G3810TL" or a single name?

Reply:
single name